assets/img/list-bullet-nested.svg

```diff
@@ -0,0 +1,9 @@
loEVqCLkWr0Dp0B9qEHkPPoVfRTvQZ2ocBTAtjYpaYC+aDcbBYLA3LxGTYXKwMq8TqsEasFf7Pl7FOrAd7jxNxBs7CXeAKjsSTcD4+HZ+LL8Gr8W14E34Cv4zfx3vxzwQ6wZTgTPAjcAkTCFmEGYRSQiVhC2E/4SR8mroIb4hEIpNoT/SGT2MqMZs4i7iEuI64i3iU2EF8SOwjkUjGJGdSACmWxCPlk0pJa0k7SEdIl0hdpHdkLbIF2Y0cTk4jS8jF5ErydvJh8iXyE3I/RZdiS/GjxFIElJmUZZTNlFbKBUoXpZ+qR7WnBlATqdnUBdQqaiP1JPUO9ZWWlpaVlq/WeC2x1nytKq3dWqe17mu9p+nTnGgc2iSagraUtpV2lHaT9opOp9vRg+lp9Hz6Uno9/Tj9Hv2dNkN7pDZXW6A9T7tGu0n7kvZzHYqOrQ5bZ4pOkU6lzl6dCzo9uhRdO12OLk93rm6N7gHd67p9egy90Xqxenl6S/S2653Re6pP0rfTD9MX6Jfob9I/rv+QgTGsGRwGn7GQsZlxktFlQDSwN+AaZBuUG+w0aDfoNdQ39DBMNiw0rDE8ZNjJxJh2TC4zl7mMuYd5jflhmNkw9jDhsMXDGoddGvbWaLhRsJHQqMxol9FVow/GLOMw4xzjFcbNxndNcBMnk/EmM0zWm5w06RluMNx/OH942fA9w2+ZoqZOpvGms0w3mZ437TMzN4swk5qtNTtu1mPONA82zzZfZX7YvNuCYRFoIbZYZXHE4k+WIYvNymVVsU6wei1NLSMtFZYbLdst+63srZKsiq12Wd21plr7WGdar7Jus+61sbAZZzPbpsHmli3F1sdWZLvG9pTtWzt7uxS7H+2a7Z7aG9lz7YvsG+zvONAdghymO9Q5XHEkOvo45jiuc7zohDp5OomcapwuOKPOXs5i53XOHSMII3xHSEbUjbjuQnNhuxS4NLjcH8kcGT2yeGTzyOejbEaljVox6tSoz66errmum11vj9YfPXZ08ejW0S/dnNz4bjVuV9zp7uHu89xb3F94OHsIPdZ73PBkeI7z/NGzzfOTl7eXzKvRq9vbxjvdu9b7uo+BT5zPEp/TvgTfEN95vgd93/t5+eX77fH729/FP8d/u//TMfZjhGM2j3kYYBXAC9gY0BnICkwP/DmwM8gyiBdUF/Qg2DpYELwl+AnbkZ3N3sF+HuIaIgvZH/KW48eZwzkaioVGhJaFtofphyWFVYfdC7cKzwpvCO+N8IyYFXE0khAZFbki8jrXjMvn1nN7x3qPnTP2RBQtKiGqOupBtFO0LLp1HDpu7LiV4+7E2MZIYppjQSw3dmXs3Tj7uOlxv40njo8bXzP+cfzo+NnxpxIYCVMTtie8SQxJXJZ4O8khSZHUlqyTPCm5PvltSmhKRUrnhFET5kw4l2qSKk5tSSOlJadtSeubGDZx9cSuSZ6TSiddm2w/uXDymSkmU3KnHJqqM5U3dW86IT0lfXv6R14sr47Xl8HNqM3o5XP4a/jPBMGCVYJuYYCwQvgkMyCzIvNpVkDWyqxuUZCoUtQj5oirxS+yI7M3ZL/Nic3ZmjOQm5K7K4+cl553QKIvyZGcmGY+rXBah9RZWirtnO43ffX0XlmUbIsckU+Wt+QbwA37eYWD4gfF/YLAgpqCdzOSZ+wt1CuUFJ6f6TRz8cwnReFFv8zCZ/Fntc22nL1g9v057Dkb5yJzM+a2zbOeVzKva37E/G0LqAtyFvxR7FpcUfx6YcrC1hKzkvklD3+I+KGhVLtUVnr9R/8fNyzCF4kXtS92X7x28ecyQdnZctfyyvKPS/hLzv40+qeqnwaWZi5tX+a1bP1y4nLJ8msrglZsq9CrKKp4uHLcyqZVrFVlq16vnrr6TKVH5YY11DWKNZ1V0VUta23WLl/7sVpUfbUmpGZXrWnt4tq36wTrLq0PXt+4wWxD+YYPP4t/vrExYmNTnV1d5SbipoJNjzcnbz71i88v9VtMtpRv+bRVsrVzW/y2E/Xe9fXbTbcva0AbFA3dOybtuLgzdGdLo0vjxl3MXeW7wW7F7j9/Tf/12p6oPW17ffY27rPdV7ufsb+sCWma2dTbLGrubElt6Tgw9kBbq3/r/t9G/rb1oOXBmkOGh5Ydph4uOTxwpOhI31Hp0Z5jWccetk1tu318wvErJ8afaD8ZdfL07+G/Hz/FPnXkdMDpg2f8zhw463O2+ZzXuabznuf3/+H5x/52r/amC94XWi76XmztGNNx+FLQpWOXQy//foV75dzVmKsd15Ku3bg+6XrnDcGNpzdzb764VXCr//b8O4Q7ZXd171beM71X9y/Hf+3q9Oo8dD/0/vkHCQ9uP+Q/fPZI/uhjV8lj+uPKJxZP6p+6PT3YHd598c+Jf3Y9kz7r7yn9S++v2ucOz/f9Hfz3+d4JvV0vZC8GXi55Zfxq62uP1219cX333uS96X9b9s743bb3Pu9PfUj58KR/xkfSx6pPjp9aP0d9vjOQNzAg5cl4qq0ABgeamQnAy60A0FMBYMA9BHWi+pynEkR9NlUh8J+w+iyoEi8AGuFNuV3nHAVgNxz26iMJUG7VE4MB6u4+NDQiz3R3U3PR4ImH8G5g4JUZAKRWAD7JBgb61w0MfNoMk70JwNHp6vOlUojwbPCztxJd3rxuPvhG/g0c8X9n3OpB2wAAAM9JREFUaAXt2bENgzAUBuFHlAE8BFswGJWHpPUYrhAFSZT6oZMSIbk4Sp8UwcdPlen1ucLrUuBxWQxfAYFgCAIJBAKQXZBAIADZBQkEApBdkEAgANkFCQQCkF2QQCAA2QUJBAKQXRAAPaH/lbdti1pr+hvLssS6rmkb6fBWoPM8Y9/39HmP40jPRzv0E4M3IhAATXf+q9F7j9ZaegullJjnOW0jHd4KNNKD/novfmIgJ5BAIADZBQkEApBdkEAgANkFCQQCkF2QQCAA2QUB0Bvz3yEOKwdAhwAAAABJRU5ErkJggg=="></image>
```

Comment:
Looks like there's a PNG embedded in here. Would it make more sense for this file to just be a `.png` image, if it's not vector-based? Should be smaller file size as an image, than as base64-encoded like this.

Reply:
After reviewing with Nick, he prefers that it is not an image at all.

Reply:
> After reviewing with Nick, he prefers that it is not an image at all.

In that case, should we remove this file from the branch?